recipe/patches/0002-xfail-test.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/sklearn/manifold/tests/test_spectral_embedding.py b/sklearn/manifold/tests/test_spectral_embedding.py
+index 08147aec3..32a5c2c9b 100644
+--- a/sklearn/manifold/tests/test_spectral_embedding.py
++++ b/sklearn/manifold/tests/test_spectral_embedding.py
+@@ -436,6 +436,8 @@ def test_spectral_embedding_preserves_dtype(eigen_solver, dtype):
+     with transformers implementing `transform` while `SpectralEmbedding`
+     implements only `fit_transform`.
+     """
++    if dtype == np.float32 and eigen_solver == "lobpcg":
++        pytest.xfail("failing on 1.20.1rc1, should pass on newer releases")
```

Comment:
I don't understand this. The last scikit-learn release is 1.2.1. Also, we usually don't do RC releases in anticipation for bugfix releases.

The next bugfix release is 1.2.2 (planned in a few weeks I thinks). The next major release is 1.3.0 (planned in a few months), maybe with an 1.3.0rc1 release but not sure.

Reply:
Also I am not 100% confident that the conda-forge `rc` branch is up to date w.r.t. the `main` branch.

Reply:
1. that is a typo: it should be `1.2.1rc1`, taken from [this line](https://github.com/conda-forge/scikit-learn-feedstock/blob/rc/recipe/meta.yaml#L1) in the branch targeted by #205
2. It seems #205 was out-of-date, and the bot did not emit another PR after #208. I will close this and start again off `main`.